recipes/erddap-python/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "erddap-python" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/erddap-python-{{ version }}.tar.gz
+  sha256: d10771f4988278c8c565e82e6af4f351929c329560aac6d8bbebc639ac3923fc
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - netcdf4
+    - pandas
+    - python >=3.6
+    - requests
+    - xarray
+
+test:
+  imports:
+    - erddapClient
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/hmedrano/erddap-python
+  summary: Python erddap API client
+  description: |
+    ERDDAP is a data server that gives you a simple, consistent way to download subsets of 
+    gridded and tabular scientific datasets in common file formats and make graphs and maps.
+
+    erddap-python is a python client for the ERDDAP Restful API, it can obtain server status 
+    metrics, provides search methods, gives tabledap and griddap class objects for metadata 
+    and data access.    
+  license: MIT
+  license_file: LICENSE.txt
+  doc_url: https://hmedrano.github.io/erddap-python/
+  dev_url: https://hmedrano.github.io/erddap-python/  
+
+extra:
+  recipe-maintainers:
+    - favio
```

Comment:
Pretty sure this should be "hmedrano"?

Reply:
Yes!, you're right I missed that. Changed it in bf8e7510ca0a976f7a3c2ba499f7293925350cd6